pkg/multicluster/o11n.go

```diff
@@ -81,3 +96,29 @@ func GetClusterInfo(_ctx context.Context, k8sClient client.Client, clusterName s
 		StorageClasses:    storageClasses,
 	}, nil
 }
+
+// GetClusterMetricsFromMetricsAPI retrieves current cluster metrics based on GetNodeMetricsFromMetricsAPI
+func GetClusterMetricsFromMetricsAPI(ctx context.Context, k8sClient client.Client, clusterName string) (*ClusterUsageMetrics, error) {
+	nodeMetricsList, err := GetNodeMetricsFromMetricsAPI(ContextWithClusterName(ctx, clusterName), k8sClient)
+	if err != nil {
+		return nil, err
+	}
+	var memoryUsage, cpuUsage resource.Quantity
+	for _, nm := range nodeMetricsList.Items {
+		cpuUsage.Add(*nm.Usage.Cpu())
+		memoryUsage.Add(*nm.Usage.Memory())
+	}
+	return &ClusterUsageMetrics{
+		CPUUsage:    cpuUsage,
+		MemoryUsage: memoryUsage,
+	}, err
+}
+
+// GetNodeMetricsFromMetricsAPI retrieves current node metrics from cluster
+func GetNodeMetricsFromMetricsAPI(ctx context.Context, client client.Client) (*metricsV1beta1api.NodeMetricsList, error) {
```

Comment:
Will this function be used outside this package? If this is designed to be consumed only by GetClusterMetricsFromMetricsAPI, we can make it getNodeMetricsFromMetricsAPI.